seq_tools/validation/c685_nested_cram_fileMd5sum_and_fileSize_match.py

```diff
@@ -0,0 +1,102 @@
+# -*- coding: utf-8 -*-
+
+"""
+    Copyright (c) 2020, Ontario Institute for Cancer Research (OICR).
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Affero General Public License as published
+    by the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU Affero General Public License for more details.
+    You should have received a copy of the GNU Affero General Public License
+    along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+    Authors:
+        Junjun Zhang <junjun.zhang@oicr.on.ca>
+        Linda Xiang <linda.xiang@oicr.on.ca>
+        Edmund Su <edmund.su@oicr.on.ca>
+"""
+
+import os
+from collections import defaultdict
+from base_checker import BaseChecker
+from seq_tools.utils import calculate_md5
+from seq_tools.utils import calculate_size
+
+class Checker(BaseChecker):
+    """
+    This checker verify whether fileSize and fileMd5sum provided in metadata are correct or not. 
+    """
+
+    def __init__(self, ctx, metadata, skip=False):
+        super().__init__(
+            ctx=ctx,
+            metadata=metadata,
+            checker_name=__name__,
+            depends_on=[  # dependent checks
+                'c180_file_uniqueness',
+                'c605_all_files_accessible'
```

Comment:
Do we need to make changes to c605 accordingly to include file accessible tests for nested `original_cram` if it does have some?